### ISSUE TITLE
Missing empty line after @api doc block is not detected

### ISSUE THREAD

**Issue by floriankraemer** [spryker/code-sniffer#285]
When reviewing [this PR](https://github.com/spryker-eco/crefo-pay-api/pull/4/files#) I noticed there is a missing line between the @api and @var in some doc blocks while in others there is. I clarified this with @dereuromark and he said there should be an empty line, just like in the facade in the same PR.

Example:

```php
    /**
     * Specification:
     *  - Url for cancel API requests.
     *
     * @api
     * @var string
     */
    public const CANCEL_API_ENDPOINT = 'CREFO_PAY_API:CANCEL_API_ENDPOINT';
```
